packages/@aws-cdk/aws-ecr-assets/lib/image-asset.ts

```diff
@@ -159,6 +159,17 @@ export interface DockerImageAssetOptions extends FingerprintOptions, FileFingerp
    * @default - hash all parameters
    */
   readonly invalidation?: DockerImageAssetInvalidationOptions;
+
+  /**
+   * Enable Docker BuildKit `${file}.dockerignore` ignore files.
+   *
+   * By default, only the `.dockerignore` file is used to ignore files.
+   * If set to `true`, an ignore file named `${file}.dockerignore` will
+   * take precedence over the default `.dockerignore` file.
+   *
+   * @default false
+   */
+  readonly buildkitIgnore?: boolean;
```

Comment:
why do we need this at all? Isn't it easier to check for `file.dockerIgnore`, and if one or more is found, have it take precedence over `.dockerIgnore`?

Reply:
so that is how i started, but after perusing the [breaking changes](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#breaking-changes) section of the contributing guide, i thought this may fall into that category.  in the event a cdk user has both files in their container directory, this change would mean the docker ignore file being used would change out from under them.  this feels like an edge case, but given the emphasis in the guide i thought i'd include it.  happy to back it out though if it's deemed too much of an edge case to worry about, since it seems it would only affect convoluted configurations.

an example folder listing where this update would change the ignore file being used:
```
.dockerignore
DockerfileProd
DockerfileProd.dockerignore
```

Reply:
ohhh I see. Good catch!

Reply:
Makes sense to me, lets leave it as is.